doc/semantics-of-the-publishing-api.md

```diff
@@ -338,6 +338,26 @@ front-end application, when the content item is rendered.
 
 ---
 
+### last_edited_at
+
+Example: *2015-01-01T12:00:00Z*
+
+Required: No
+
+The `last_edited_at` records the last time that the content received a major or
+minor update. This can be used by the publishing applications as a ordering
+parameter and shown to editors.
+
+If the `last_updated_at` is specified when PUTing content, the given value will
+be stored. If `last_updated_at` is not specified, and the update type is major
+or minor, the `last_updated_at` will be set to the current time. For any other
+update type, the `last_updated_at` will be unchanged.
```

Comment:
Currently `put_content` looks to process this fine. I don't think this should be the case, but the correct place to stop this is at the schema validation level, which is before any of the logic associated with processing `last_edited_at`?